docs/css/background-position.md

```diff
@@ -0,0 +1,90 @@
+> Backgrounds
+
+# Background Position
+
+Utilities for controlling the position of an element's background image.
+
+## Quick reference
+
+<qr-table />
+
+## Basic usage
+
+### Setting the background position
+Use the `bg-{side}` utilities to control the position of an element’s background image.
+
+<example-container>
+  <div class="grid sm:grid-cols-3 gap-24 sm:justify-around">
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-left-top</div>
+      <div class="h-128 w-128 rounded-8 bg-top-left bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-top</div>
+      <div class="h-128 w-128 rounded-8 bg-top bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-right-top</div>
+      <div class="h-128 w-128 rounded-8 bg-top-right bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-left</div>
+      <div class="h-128 w-128 rounded-8 bg-left bg-auto bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-center</div>
+      <div class="h-128 w-128 rounded-8 bg-center bg-auto bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-right</div>
+      <div class="h-128 w-128 rounded-8 bg-right bg-auto bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
+    </div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="pb-8 pd-font-mono">bg-left-bottom</div>
+      <div class="h-128 w-128 rounded-8 bg-bottom-left bg-auto bg-[url(./50s-scientists.jpg)]" style="background-size:300%"></div>
```

Comment:
<img width="759" alt="image" src="https://github.com/warp-ds/tech-docs/assets/462825/5369283b-bc7d-4687-bc0b-6a172fc2287e">
2fc2287e">

I dont see why but some of the classes like the bg-bottom-left and bg-bottom-right are not getting any styles generated , maybe it turned arround, is it bg-bottom-left, or bg-left-bottom ?

Reply:
The code example was correct, but the visual example where mixed up. 😅  Fixed now. Thanks. 👍